src/sentry/eventstore/snuba/backend.py

```diff
@@ -160,9 +159,7 @@ def __get_event_id_from_filter(self, filter=None, orderby=None):
             limit=1,
             referrer="eventstore.get_next_or_prev_event_id",
             orderby=orderby,
-            dataset=snuba.detect_dataset(
-                {"conditions": filter.conditions}, aliased_conditions=True
-            ),
+            dataset=snuba.detect_dataset({"conditions": filter.conditions}),
```

Comment:
Note that this changes the dataset detection logic to not take into account any selected columns. Intentional?

Reply:
The `event_id` and `project_id` columns are present on all datasets.

Reply:
Sorry, I misread the diff and thought the was in get_events